### ISSUE TITLE
Improve lore2sphinx buildbot results for database howtos in `projects/core/howto/`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4871]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4871|
|Type|enhancement|
|Created|2011-02-11 05:34:02Z|
|Branch|https://github.com/twisted/twisted/tree/db-sphinx-4871|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the row and rdbms howtos

<details><summary>Searchable metadata</summary>

```
trac-id__4871 4871
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_db_sphinx_4871 branches/db-sphinx-4871
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402442000000 1297402442000000
changetime__1300075645000000 1300075645000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30674]) Branching to 'db-sphinx-4871'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30675]) Merge the database parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs [#4871](https://github.com/twisted/twisted/issues/4871)
refs #4568

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_5"></a>|@washort set owner to @exarkun|
|-|-|

Success. Markup renders right in Sphinx and in Lore.

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_6"></a>|@washort set status to `closed`|
|-|-|

(In [31002]) Merge changes to rdbms.xhtml and row.xhtml from db-sphinx-4871.

Author: screwtape, jdb
Reviewer: washort
Fixes: [#4871](https://github.com/twisted/twisted/issues/4871)

Fixes formatting changes needed to make sphinx happy with rdbms.xhtml and row.xhtml.
